map-widget/qml/MapWidget.qml

```diff
@@ -15,7 +15,7 @@ Item {
 		id: mapHelper
 		map: map
 		editMode: false
-		onSelectedDivesChanged: rootItem.selectedDivesChanged(list)
+		onSelectedDivesChanged: (list) => { rootItem.selectedDivesChanged(list) }
```

Comment:
I'm 99% certain that I didn't need that change when getting things to work with Qt6
is this a new requirement that handlers need to be functions instead of just a line of code? When did that start, do you know?